src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/BrevModul/Brev.tsx

```diff
@@ -15,12 +16,9 @@ interface IProps {
 const Brev = ({ onOkIModalClick }: IProps) => {
     const { åpenBehandling } = useBehandling();
     const { fagsak } = useFagsakRessurser();
-    const {
-        hentForhåndsvisning,
-        hentetForhåndsvisning,
-        hentMuligeBrevMaler,
-        navigerTilOpplysningsplikt,
-    } = useBrevModul();
+    const { hentMuligeBrevMaler, navigerTilOpplysningsplikt } = useBrevModul();
+    const { hentForhåndsvisning, hentetForhåndsvisning } = useForhåndsvisning();
```

Comment:
Kan dette ligge direkte i Brevskjema?

Reply:
Det skulle nog det, nå er denne variablen ikke en del av denne oppgaven og kanske har den som gjort det tenkt at man skal kunne ta forskjellige set av brevmaler. Men ettersom dette ikke er en del av denne oppgaven og om det er ok for deg velger jeg å ikke endre på nåværende implementasjon.

Reply:
Det ble gjort fordi man ikke hadde useForhåndsvisning. Det er fint om det kan forbedres, slik at løsningen kontinuerlig blir bedre og vi risikerer mindre teknisk gjeld senere :)

Reply:
Ok, fikser det.